ext/pdo_mysql/tests/gh15432.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
```

Comment:
Since we are testing mysqlnd extension, one test is enough. You can remove the PDO one. PDO can be built against libmysql so we could run into some issues too, e.g. libmysql version.